packages/selenium/test/RunPartial/FrameTests.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections.Generic;
+using System.Linq;
+using Deque.AxeCore.Commons;
+using Deque.AxeCore.Selenium;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class FrameTests : TestBase
+    {
+        [Test]
+        public void ShouldAnalyzeNestedIFrames()
+        {
+            GoToFixture("nested-iframes.html");
+
+            var res = new AxeBuilder(driver)
+                .WithOptions(new AxeRunOptions
```

Comment:
nit: can be written more simply as `.WithRules("label")` (same for other tests in file)